linbox/matrix/densematrix/blas-submatrix.h

```diff
@@ -0,0 +1,347 @@
+/*
+ * Copyright (C) 2004 Pascal Giorgi, Clément Pernet
+ *               2013, 2014 the LinBox group
+ *               2018 revamped by Pascal Giorgi 
+ * Written by :
+ *               Pascal Giorgi  pascal.giorgi@lirmm.fr
+ *               Clément Pernet clement.pernet@imag.fr
+ *               Brice Boyer (briceboyer) <boyer.brice@gmail.com>
+ *
+ * ========LICENCE========
+ * This file is part of the library LinBox.
+ *
+ * LinBox is free software: you can redistribute it and/or modify
+ * it under the terms of the  GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.	 See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ * ========LICENCE========
+ */
+
+/*! @file matrix/densematrix/new-blas-submatrix.h
+ * @ingroup densematrix
+ */
+
+#ifndef __LINBOX_densematrix_blas_submatrix_H
+#define __LINBOX_densematrix_blas_submatrix_H
+
+
+#include "linbox/matrix/MatrixDomain/apply-domain.h"
```

Comment:
I fail compiling, `MatrixDomain` should be lowercase.

Reply:
fixed.